phpBB/phpbb/files/filespec.php

```diff
@@ -394,7 +394,8 @@ public function check_content($disallowed_content)
 	 *
 	 * @param string $destination Destination path, for example $config['avatar_path']
 	 * @param bool $overwrite If set to true, an already existing file will be overwritten
-	 * @param bool $skip_image_check If set to true, the check for the file to be a valid image is skipped
+	 * @param bool $skip_image_check If set to true, the check for the file to
+	 *                               be a valid image is skipped
```

Comment:
Unrelated change?

Reply:
I don't know where it is coming from, I didn't touch the file....